robottelo/test.py

```diff
@@ -238,11 +240,21 @@ def take_screenshot(self, testmethodname):
         testmethodname = "_".join(testmethodname)
         # Creates dir_structure depending upon the testmethodname.
         directory = testmethodname
-        if not os.path.exists("/tmp/screenshots/" + directory):
-            os.makedirs("/tmp/screenshots/" + directory)
-        self.browser.save_screenshot('/tmp/screenshots/{dir}/'
-                                     'screenshot-{date}.png'
-                                     .format(dir=directory, date=filename))
+        os_ver = self.server_name.split(".")[0].split("-")[2]
```

Comment:
We can not assume this, because we can have a server name like `ibm-something.domain.com` which does not contains the os version on it. I think for the directory name should be formed by just `{base_dir}/{date}/{dir}`